YAML config loader

ubawurinna opened the pull request:
Uses the same loading method for the go2rtc configuration as for the main configuration

Commit:
Uses the same loading method for the go2rtc configuration as for the main configuration

NickM-27 commented:
I'm not sure I see the benefit to this, the main goal here is to load the go2rtc config, we don't care if the frigate config is valid and IMO it's preferable for it to fail on Frigate load since that gives more information in the logs compared to here

Commit:
Merge branch 'dev' into yaml_loader

Commit:
Uses PyYAML's implicit resolver to handle all variable substitutions when loading the YAML file.
Environment variables must use ${FRIGATE } format (add $ symbol) and must not be enclosed in quotation marks.
